tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check yast-timesync.timer", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
+    my $conf_file = script_output("cat /etc/systemd/system/yast-timesync.timer");
+    $conf_file =~ /OnUnitActiveSec=(?<interval>(\S+))min/;
+    my $expected_interval = $test_data->{profile}->{'ntp-client'}->{ntp_sync};
+    assert_equals($expected_interval, $+{interval}, "The interval in yast-timesync configuration file is not the expected one.");
+    # Default value for first time sync is 1 minute. Module pauses in order to be able to track the sync in the logs.
+    sleep 60;
+    record_info("Check logs for timesync", "Check if the use of systemd timers can be spotted in the journal logs as it's expected.");
+    assert_script_run('journalctl -u yast-timesync --since "2 minutes ago" | grep "Started One time sync configured by YaST"');
+    record_info("Check time deviation correction", "Check if \"One time synchronization\" will fix time deviation as expected.");
+    script_run("date");
+    assert_script_run('date $(date +%m%d%H%M%Y.%S -d "1 minute ago")');
+    assert_script_run('journalctl -fu yast-timesync --since now | grep "wrong by"', ($expected_interval * 60));
+    my $time_deviation = script_output('journalctl -fu yast-timesync | grep -oP "(?<=wrong by\s)[0-9,.,-]+"');
```

Comment:
I tried 
my $time_deviation = script_output('journalctl -fu yast-timesync | grep -oP "(?<=wrong by\s)[0-9,.,-]+"', ($expected_interval * 60));

without having the assert_script_run above it, but didn't work. Maybe there is some issue with script_output.

Reply:
Also, increased interval from 2 to 3 minutes, to avoid having the sync while commands of line 36,37 are being typed.

Reply:
> I tried
> my $time_deviation = script_output('journalctl -fu yast-timesync | grep -oP "(?<=wrong by\s)[0-9,.,-]+"', ($expected_interval * 60));
> 
> without having the assert_script_run above it, but didn't work. Maybe there is some issue with script_output.

maybe the `-f` does not work, because that option monitor the output but not ret code for openqa...

Reply:
Yep.. I tried a lot to make this work. Basically I thought that once the script is asserted it would stop, despite the $wait.  Now, I replaced this with a while loop, so that there is no chance of sporadic failure.